src/render.js

```diff
@@ -424,34 +423,25 @@ class App extends Preact.Component {
     const isOfMimeType = (type) => (entry) => entry._type === type;
 
     // No input - return everything
-    if (!input) return unfiltered;
+    if (!search) return unfiltered;
 
     // No label - search by text (default)
-    if (!input.includes(":"))
-      return (entry) => isOfType("text")(entry) && entry.value.includes(input);
+    if (!search.includes(":"))
+      return (entry) => isOfType("text")(entry) && entry.value.includes(search);
     else {
-      const [label, value] = input
-        ?.split(":") // split on ":"
-        ?.map((element) => element.trim()); // trim whitespaces
+      // Splitting the string on ":" to get label and value (search = label: value)
+      const [label, value] = search
+        ?.split(":")
+        ?.map((element) => element.trim());
 
       if (label === "text")
         //TODO improve searching
         return (entry) =>
           isOfType("text")(entry) && entry.value.includes(value);
 
-      if (label === "image") {
-        //TODO more MIME types
-        if (value === "image/png")
-          return (entry) =>
-            isOfType("image")(entry) && isOfMimeType("image/png")(entry);
-
-        if (value === "image/jpeg" || value === "image/jpg")
-          return (entry) =>
-            isOfType("image")(entry) && isOfMimeType("image/jpeg")(entry);
-
-        // Provided MIME type did't match any case - return every image
-        return (entry) => isOfType("image")(entry);
-      }
+      if (label === "image")
+        return (entry) =>
+          isOfType("image")(entry) && isOfMimeType(value)(entry);
```

Comment:
This would require the user to type in the full MIME type, yeah?

Reply:
yes

Reply:
I can add `|| isOfMimeType(`image/${value}`)(entry)`

Reply:
then image/png and png would work

Reply:
How about we allow the users a more friendly experience? So instead of typing the full `image: image/jpeg` MIME type, they just type in `image: jpeg` like suggested previously? 😃

Reply:
Oh, replied after I got your responses. Sure thing, that'd be more accurate and friendly at the same time! Nice!